gatsby-browser.js

```diff
@@ -8,11 +8,17 @@
 
 import React from "react"
 import { ThemeProvider } from "emotion-theming"
+import { Global } from "@emotion/core"
 
 import theme from "./src/theme"
-import "./src/reset.css"
+import globalStyles from "./src/global"
 
 // eslint-disable-next-line import/prefer-default-export
 export const wrapRootElement = ({ element }) => (
-  <ThemeProvider theme={theme}>{element}</ThemeProvider>
+  <>
+    <ThemeProvider theme={theme}>
+      <Global styles={globalStyles} />
```

Comment:
These styles will be read on runtime on SSR i think <link rel="stylesheet" href="globalStyles.min.css" /> is better and lesser CPU usage.

Reply:
The problem though FOUC. So forget about my comment